airbyte-server/src/main/java/io/airbyte/server/apis/WorkspaceApiController.java

```diff
@@ -15,63 +15,76 @@
 import io.airbyte.api.model.generated.WorkspaceUpdate;
 import io.airbyte.api.model.generated.WorkspaceUpdateName;
 import io.airbyte.server.handlers.WorkspacesHandler;
-import javax.ws.rs.Path;
-import lombok.AllArgsConstructor;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Post;
 
-@Path("/v1/workspaces")
-@AllArgsConstructor
+@Controller("/api/v1/workspaces")
 public class WorkspaceApiController implements WorkspaceApi {
 
   private final WorkspacesHandler workspacesHandler;
 
+  public WorkspaceApiController(final WorkspacesHandler workspacesHandler) {
+    this.workspacesHandler = workspacesHandler;
+  }
+
+  @Post("/create")
```

Comment:
Should we add the payload mediatype?

Reply:
The `@Controller` annotation defaults to `application/json`, so we only need to specify it if we're not using json.